src/views/Expenses.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+
+  // Run on initial render only
+  useEffect(() => {
+    // Fetch expense items
+    const token = process.env.REACT_APP_MOSTASH_API_KEY;
```

Comment:
maybe we also save the hostname in the env

Reply:
Good idea!